Sprint-1/1-key-exercises/4-random.js

```diff
@@ -7,3 +7,18 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+
+// num represents the result of equation at the right hand side of the operator =
+
+// Step-1 ---> Math.random() generates a random decimal,
+// Step-2 ---> (maximum-minumum +1) is evaluated 
+// Step-3 ---> Math.random() is multiplied by (maximum-minumum +1)
+// Step-4 ---> the resukt of Step-3 is round by the method Math.floor
+// Step-5 ---> minimum is added to the result of Step-4
+
+// Thus, num represents random numbers generated between minimum 1 and maximum 100. As we run the code several times it
```

Comment:
`Thus,` suggests to me this _might've_ been generated by AI

Remember, the aim of this is to ensure you're understanding the exercises. It's hard to do this if you don't write the answers in your own words